challenge.js

```diff
@@ -58,4 +63,5 @@
 
 // ANSWER
 
+Node.appendChild()
```

Comment:
Correct. All HTML elements are Nodes; if `div` is a reference to an on-DOM div, and we have a off-DOM element `e` we created in our JS, we can attach it with this method:

```js
var div = document.querySelectorAll("body > div")[0]; // the zeroth element of the set of all divs that are immediate children of the body
var e = document.createElement('textarea'); // this textarea element is not on the DOM yet
div.appendChild(e); // mission accomplished
```

1